src/js/actions/type.js

```diff
@@ -240,20 +241,21 @@ define(function (require, exports) {
      * @param {Document} document
      * @param {Immutable.Iterable.<Layers>} layers
      * @param {Color} color
-     * @param {boolean=} coalesce Whether to coalesce this operation's history state
-     * @param {boolean=} ignoreAlpha Whether to ignore the alpha value of the
+     * @param {boolean=} options.coalesce Whether to coalesce this operation's history state
+     * @param {boolean=} options.ignoreAlpha Whether to ignore the alpha value of the
      *  given color and only update the opaque color value.
      * @return {Promise}
      */
-    var setColor = function (document, layers, color, coalesce, ignoreAlpha) {
+    var setColor = function (document, layers, color, options) {
         var layerIDs = collection.pluck(layers, "id"),
             layerRefs = layerIDs.map(textLayerLib.referenceBy.id).toArray(),
             normalizedColor = color.normalizeAlpha(),
             opaqueColor = normalizedColor.opaque(),
             playObject = textLayerLib.setColor(layerRefs, opaqueColor),
-            typeOptions = _getTypeOptions(document.id, strings.ACTIONS.SET_TYPE_COLOR, coalesce);
+            typeOptions = _.merge(options,
```

Comment:
sorry to beat a dead horse, but...
the weird side effect of this is that the descriptor ALSO gets the "ignoreAlpha" property which is meaningless.  I think a more "strict" approach would be to pass the `options` in to _getTypeOptions as a parameter and "cherry pick" the props into their appropriate places in the final typeOptions that will be sent to the descriptor (coalesce, transaction).   

basically, I'm worried that there isn't a clear delineation between the options that gets passed in to our actions, and those that get passed in to descriptor.  Maybe I'm overreacting. 

Related: transaction should probably be specified in JSDoc if that's a meaningful option to `setColor`.